docs/tutorials/onboarding_videos.md

```
@@ -0,0 +1,14 @@
+# Onboarding videos
+
+This page includes some basic videos showcasing where to find the certain POIs (Points Of Interest) regarding Formula Trinity Autonomous
+ 
+### Finding the FT-Autonomous Locker and the "Rigby-Room":
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)